apps/evm/src/clients/api/queries/getProposalEta/index.ts

```diff
@@ -10,13 +12,19 @@ export type GetProposalEtaOutput = {
 };
 
 const getProposalEta = async ({
-  governorBravoDelegateContract,
+  publicClient,
+  governorBravoDelegateContractAddress,
   proposalId,
 }: GetProposalEtaInput): Promise<GetProposalEtaOutput> => {
-  const resp = await governorBravoDelegateContract.proposals(proposalId);
+  const [_id, _proposerAddress, etaSeconds] = await publicClient.readContract({
+    address: governorBravoDelegateContractAddress,
+    abi: governorBravoDelegateAbi,
+    functionName: 'proposals',
+    args: [BigInt(proposalId)],
+  });
 
   // Convert ETA expressed in seconds to milliseconds
-  const eta = new Date(+resp.eta * 1000);
+  const eta = new Date(Number(etaSeconds) * 1000);
```

Comment:
I like this change, `+string` to turn a string into a number is just JS wizardry

Reply:
Hehe sometimes I remember that some primitives can be used as functions.